pages/[user]/comments.js

```diff
@@ -0,0 +1,144 @@
+import {shallowEqual, useSelector} from 'react-redux';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+
+import { getIsomorphicDataPopulation } from '../../store';
+import { loadUserCommentsPage } from '../../store/actions';
+import Userpic from '../../components/Userpic';
+import UserFeedStatus from '../../components/UserFeedStatus';
+import DummyPost from '../../components/DummyPost';
+import PieceOfText from '../../components/PieceOfText';
+import Post from '../../components/Post';
+import PaginationLinks from '../../components/PaginationLinks';
+
+const CommentedPostsPage = () => {
+  const { query: { user: username } } = useRouter();
+  const isLoadingPage = useSelector(state => state.isLoadingPage);
+  const user = useSelector(state => Object.values(state.users).find(u => u.username === username));
+  const postIds = useSelector(state => Object.keys(state.posts), shallowEqual);
+
+  if (isLoadingPage) {
+    return (
+      <main>
+        <h1>{user?.displayName || username}</h1>
+        <DummyPost/>
+        <DummyPost/>
+        <DummyPost/>
+      </main>
+    );
+  }
+
+  if (!user) {
+    return (
+      <main>
+        <h1>{username}</h1>
+        <p>User not found</p>
+      </main>
+    );
+  }
+
+  return (
+    <main>
+      <div className="info">
+        <section className="userpic">
+          <Link href={`/${username}`}>
+            <a><Userpic id={user.id} size={75}/></a>
+          </Link>
+        </section>
+        <section className="user-data">
+          <h1>{user.displayName}</h1>
+          <div className="username">
+            @{username}
+          </div>
+          <div className="user-description">{user.description && <><PieceOfText isExpanded>{user.description}</PieceOfText></>}</div>
+        </section>
+      </div>
+
+      {user.statistics && (
+        <p className="statistics">
+          <a>{user.statistics.subscribers} subscribers</a>{' '}
+
+          {user.type === 'user' && <>
+            <a>{user.statistics.subscriptions} subscriptions</a>
+            {' '}
+            <Link href={`/${username}`}>
+              <a>{user.statistics.posts} posts</a>
+            </Link>
+            {' '}
+            <Link href={`/${username}/comments`}>
+              <a>{user.statistics.comments} comments</a>
+            </Link>
+            {' '}
+            <Link href={`/${username}/likes`}>
+              <a>{user.statistics.likes} likes</a>
+            </Link>
+          </>}
+        </p>
+      )}
+
+      <p className="statuses">
+        <UserFeedStatus {...user}/>
+      </p>
+
+      <p className="feedHeader">
+          Comments
+      </p>
```

Comment:
I'd prefer to use semantic element, such as &lt;h2&gt;, instead of another &lt;p&gt;. (Also, check the indentation.)